src/trunk.c

```diff
@@ -143,15 +156,19 @@ trunk_close_log_stream_if_enabled(trunk_handle           *spl,
 #define trunk_log_stream_if_enabled(spl, _stream, message, ...)                \
    do {                                                                        \
       if (trunk_verbose_logging_enabled(spl)) {                                \
-         platform_log_stream(                                                  \
-            (_stream), "[%3lu] " message, platform_get_tid(), ##__VA_ARGS__);  \
+         platform_log_stream((_stream),                                        \
+                             "trunk_log():%d [%lu] " message,                  \
+                             __LINE__,                                         \
+                             platform_get_tid(),                               \
+                             ##__VA_ARGS__);                                   \
       }                                                                        \
    } while (0)
 
 #define trunk_default_log_if_enabled(spl, message, ...)                        \
    do {                                                                        \
       if (trunk_verbose_logging_enabled(spl)) {                                \
-         platform_default_log(message, __VA_ARGS__);                           \
+         platform_default_log(                                                 \
+            "trunk_log():%d " message, __LINE__, __VA_ARGS__);                 \
```

Comment:
Here, and on L161 above, print the line # where this trace message is generated from, which helps quickly spotting the code which got executed.